packages/cli/generators/model/index.indexed-props.js.bak

```diff
@@ -0,0 +1,262 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
```

Comment:
The file name says `packages/cli/generators/model/index.indexed-props.js.bak`, is this a temporary backup you forgot to remove?